webrtc/README.md

```diff
@@ -67,6 +67,10 @@ message Message {
     // The sender abruptly terminates the sending part of the stream. The
     // receiver MAY discard any data that it already received on that stream.
     RESET_STREAM = 2;
+    // The receiver has previously sent a message with a FIN flag. On receiving
+    // the FIN_ACK flag they may now close the stream as they know the remote has received
```

Comment:
```suggestion
    // the FIN_ACK flag they may now close the write-side of the stream as they know the remote has received
```

We should still continue reading, right?

Reply:
RTCDataChannels don't support half-closed streams, the whole thing is either open or closed, so saying "close the write-side" could be a bit confusing?

Reply:
This is refering to our framing mechanism which supports half-closing, right?

Reply:
Ah, ok - this is my mistake then, the idea here is this is the datachannel rather than the libp2p stream.  It should probably say that..

Reply:
Unfortunately, the browser APIs don't give us access to the SCTP APIs which would support all of these things. Hence we needed to make our own framing. AFAIK, all we can do on a data channel is send data and close it entirely. No half-closing, no resetting.

Reply:
Indeed, yes.

Anyway what I meant was it should probably say:

```suggestion
    // the FIN_ACK flag they may now close the datachannel as they know the remote has received
```

Reply:
I don't think this is true, right? The remote is just acknowledging that _we_ closed our side of the stream. Unless the remote has also sent us a `FIN`, we need to keep reading.

Reply:
It's true if the remote delays sending the `FIN_ACK` until it's received a `FIN_ACK`, otherwise yes, it's not enough to allow closing the channel.  I think this is hard to see on first reading so the language should be clearer.

I've removed the text that implies what you can do after receiving the flag, leaving that to the "Closing an `RTCDataChannel`" section.